ecosystem/sep-0006.md

```diff
@@ -1010,6 +1011,22 @@ Name | Type | Description
 * `too_large` -- deposit/withdrawal size exceeded `max_amount`.
 * `error` -- catch-all for any error not enumerated above.
 
+### Refund Object Schema
+
+Name | Type | Description
+-----|------|------------
+`type` | string | One of `partial` or `full`
```

Comment:
I think we can drop this field, since it overlaps with the existing top-level `refunded` field. The top-level refunded field could indicate if the transaction has been fully refunded or not, and the presence of any refunds in the list will indicate that the transaction is partially refunded.

Reply:
I'm not a huge fan of this because I can see a client misinterpreting `refunded: false` to mean that there were no refund payments whatsoever. 

The SEP currently doesn't define the concept of a partial refund, so any client that checks the `refunded` attribute today almost certainly interprets `refunded: false` to mean no value was refunded. From that perspective, we'd be redefining what `refunded: false` means, which I think is more problematic than deprecating the field.

Reply:
The type field may be more descriptive, but since we have another field that already communicates the same thing it feels like we're improving something that still carries the same cost of continuing to use the thing we already have.

Reply:
Clients should be able to figure out if this is a total or partial refund by comparing `refunds.amount_refunded` with `amount_in` like you described in the field `amount_refunded` below.

Reply:
Oh, thats a good point, I'll remove it.